### PR TITLE
Add extra attribute in `PrintOp` to propagate signness info

### LINKED ISSUE #1705: [UT] Fix `test_subprocess.py::test_print[device_print_negative-int32]`
```
Expected line "pid (0, 0, 0) idx (  1) x: 4294967295" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx (  2) x: 4294967294" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx (  3) x: 4294967293" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx (  4) x: 4294967292" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx (  5) x: 4294967291" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx (  6) x: 4294967290" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx (  7) x: 4294967289" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx (  8) x: 4294967288" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx (  9) x: 4294967287" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 10) x: 4294967286" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 11) x: 4294967285" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 12) x: 4294967284" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 13) x: 4294967283" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 14) x: 4294967282" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 15) x: 4294967281" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 16) x: 4294967280" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 17) x: 4294967279" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 18) x: 4294967278" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 19) x: 4294967277" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 20) x: 4294967276" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 21) x: 4294967275" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 22) x: 4294967274" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 23) x: 4294967273" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 24) x: 4294967272" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 25) x: 4294967271" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 26) x: 4294967270" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 27) x: 4294967269" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 28) x: 4294967268" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 29) x: 4294967267" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 30) x: 4294967266" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 31) x: 4294967265" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 64) x: 4294967232" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 65) x: 4294967231" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 66) x: 4294967230" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 67) x: 4294967229" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 68) x: 4294967228" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 69) x: 4294967227" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 70) x: 4294967226" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 71) x: 4294967225" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 72) x: 4294967224" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 73) x: 4294967223" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 74) x: 4294967222" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 75) x: 4294967221" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 76) x: 4294967220" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 77) x: 4294967219" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 78) x: 4294967218" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 79) x: 4294967217" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 80) x: 4294967216" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 81) x: 4294967215" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 82) x: 4294967214" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 83) x: 4294967213" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 84) x: 4294967212" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 85) x: 4294967211" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 86) x: 4294967210" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 87) x: 4294967209" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 88) x: 4294967208" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 89) x: 4294967207" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 90) x: 4294967206" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 91) x: 4294967205" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 92) x: 4294967204" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 93) x: 4294967203" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 94) x: 4294967202" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 95) x: 4294967201" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 32) x: 4294967264" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 33) x: 4294967263" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 34) x: 4294967262" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 35) x: 4294967261" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 36) x: 4294967260" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 37) x: 4294967259" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 38) x: 4294967258" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 39) x: 4294967257" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 40) x: 4294967256" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 41) x: 4294967255" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 42) x: 4294967254" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 43) x: 4294967253" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 44) x: 4294967252" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 45) x: 4294967251" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 46) x: 4294967250" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 47) x: 4294967249" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 48) x: 4294967248" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 49) x: 4294967247" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 50) x: 4294967246" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 51) x: 4294967245" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 52) x: 4294967244" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 53) x: 4294967243" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 54) x: 4294967242" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 55) x: 4294967241" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 56) x: 4294967240" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 57) x: 4294967239" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 58) x: 4294967238" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 59) x: 4294967237" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 60) x: 4294967236" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 61) x: 4294967235" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 62) x: 4294967234" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 63) x: 4294967233" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 96) x: 4294967200" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 97) x: 4294967199" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 98) x: 4294967198" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx ( 99) x: 4294967197" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx (100) x: 4294967196" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx (101) x: 4294967195" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx (102) x: 4294967194" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx (103) x: 4294967193" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx (104) x: 4294967192" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx (105) x: 4294967191" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx (106) x: 4294967190" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx (107) x: 4294967189" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx (108) x: 4294967188" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx (109) x: 4294967187" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx (110) x: 4294967186" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx (111) x: 4294967185" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx (112) x: 4294967184" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx (113) x: 4294967183" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx (114) x: 4294967182" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx (115) x: 4294967181" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx (116) x: 4294967180" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx (117) x: 4294967179" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx (118) x: 4294967178" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx (119) x: 4294967177" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx (120) x: 4294967176" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx (121) x: 4294967175" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx (122) x: 4294967174" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx (123) x: 4294967173" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx (124) x: 4294967172" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx (125) x: 4294967171" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx (126) x: 4294967170" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx (127) x: 4294967169" 0 time(s), but saw 1 time(s)
Expected line "pid (0, 0, 0) idx (  1) x: -1" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx (  2) x: -2" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx (  3) x: -3" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx (  4) x: -4" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx (  5) x: -5" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx (  6) x: -6" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx (  7) x: -7" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx (  8) x: -8" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx (  9) x: -9" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 10) x: -10" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 11) x: -11" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 12) x: -12" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 13) x: -13" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 14) x: -14" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 15) x: -15" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 16) x: -16" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 17) x: -17" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 18) x: -18" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 19) x: -19" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 20) x: -20" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 21) x: -21" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 22) x: -22" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 23) x: -23" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 24) x: -24" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 25) x: -25" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 26) x: -26" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 27) x: -27" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 28) x: -28" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 29) x: -29" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 30) x: -30" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 31) x: -31" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 32) x: -32" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 33) x: -33" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 34) x: -34" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 35) x: -35" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 36) x: -36" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 37) x: -37" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 38) x: -38" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 39) x: -39" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 40) x: -40" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 41) x: -41" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 42) x: -42" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 43) x: -43" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 44) x: -44" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 45) x: -45" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 46) x: -46" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 47) x: -47" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 48) x: -48" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 49) x: -49" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 50) x: -50" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 51) x: -51" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 52) x: -52" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 53) x: -53" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 54) x: -54" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 55) x: -55" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 56) x: -56" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 57) x: -57" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 58) x: -58" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 59) x: -59" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 60) x: -60" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 61) x: -61" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 62) x: -62" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 63) x: -63" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 64) x: -64" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 65) x: -65" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 66) x: -66" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 67) x: -67" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 68) x: -68" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 69) x: -69" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 70) x: -70" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 71) x: -71" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 72) x: -72" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 73) x: -73" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 74) x: -74" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 75) x: -75" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 76) x: -76" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 77) x: -77" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 78) x: -78" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 79) x: -79" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 80) x: -80" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 81) x: -81" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 82) x: -82" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 83) x: -83" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 84) x: -84" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 85) x: -85" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 86) x: -86" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 87) x: -87" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 88) x: -88" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 89) x: -89" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 90) x: -90" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 91) x: -91" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 92) x: -92" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 93) x: -93" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 94) x: -94" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 95) x: -95" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 96) x: -96" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 97) x: -97" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 98) x: -98" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx ( 99) x: -99" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx (100) x: -100" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx (101) x: -101" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx (102) x: -102" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx (103) x: -103" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx (104) x: -104" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx (105) x: -105" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx (106) x: -106" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx (107) x: -107" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx (108) x: -108" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx (109) x: -109" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx (110) x: -110" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx (111) x: -111" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx (112) x: -112" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx (113) x: -113" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx (114) x: -114" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx (115) x: -115" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx (116) x: -116" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx (117) x: -117" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx (118) x: -118" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx (119) x: -119" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx (120) x: -120" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx (121) x: -121" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx (122) x: -122" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx (123) x: -123" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx (124) x: -124" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx (125) x: -125" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx (126) x: -126" 1 time(s), but saw 0 time(s)
Expected line "pid (0, 0, 0) idx (127) x: -127" 1 time(s), but saw 0 time(s)
```

### PR DESCRIPTION
**Pull request by anmyachev**
Applies changes from https://github.com/triton-lang/triton/commit/a94dee0cfd213105831dc91c3d2236856e248035 to intel-specific code.

Closes #1705

### CODE REVIEW

File: third_party/intel/lib/TritonIntelGPUToLLVM/PrintOpToLLVM.cpp [hunk 1]
```diff
@@ -49,7 +49,10 @@ struct PrintOpConversion
       LLVM::intel::llPrintf(rewriter, formatStr,
                             {pid[0], pid[1], pid[2], prefixStr});
     } else {
+      assert(op.getNumOperands() == op.getIsSigned().size());
```
Comment (etiotto) [inline third_party/intel/lib/TritonIntelGPUToLLVM/PrintOpToLLVM.cpp:52]: add an assert message pls.

Comment (anmyachev) [reply third_party/intel/lib/TritonIntelGPUToLLVM/PrintOpToLLVM.cpp]: done

**Review by whitneywhtsang — APPROVED** [review]

**Comment by whitneywhtsang** [comment]
@anmyachev Please check if you are ok with changes in e150be30e3654cab9de5532a2d4524a6842d6a08.

**Comment by anmyachev** [comment]
> @anmyachev Please check if you are ok with changes in [e150be3](https://github.com/intel/intel-xpu-backend-for-triton/commit/e150be30e3654cab9de5532a2d4524a6842d6a08).

@whitneywhtsang looks good! Thank you!

**Pull request merged**